manager/controlapi/secret.go

```diff
@@ -238,16 +238,26 @@ func (s *Server) RemoveSecret(ctx context.Context, request *api.RemoveSecretRequ
 	}
 }
 
+// ValidateSecretPayload validates the secret payload size
+func ValidateSecretPayload(data []byte) error {
```

Comment:
I'd prefer to move this to a subpackage like `api/secret` or `api/validation`, so that dispatcher doesn't import `controlapi`.